manifest/provider/provider_config.go

```diff
@@ -54,6 +54,17 @@ func GetProviderConfigSchema() *tfprotov5.Schema {
 				DescriptionKind: 0,
 				Deprecated:      false,
 			},
+			{
+				Name:            "tls_server_name",
+				Type:            tftypes.String,
+				Description:     "Passed to the server for SNI and is used in the client to check server certificates against.",
```

Comment:
This needs to match exactly what's in the schema for the main provider above, so this needs to be:

```
Server name passed to the server for SNI and is used in the client to check server certificates against.
```

It's not super intuitive, but when you run the provider in debug mode you can see the failure in the logs:

```
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64: 2022/03/21 12:07:35 got a different provider schema from two servers (*schema.GRPCProviderServer, *provider.RawProviderServer). Provider schemas must be identical across providers. Diff:   &tfprotov5.Schema{
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   	Version: 0,
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   	Block: &tfprotov5.SchemaBlock{
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		Version: 0,
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		Attributes: []*tfprotov5.SchemaAttribute(Inverse(cmpopts.SortSlices, []*tfprotov5.SchemaAttribute{
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		... // 10 identical elements
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		&{Name: "password", Type: s"tftypes.String", Description: "The password to use for HTTP basic authentication when accessing"..., Optional: true, ...},
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		&{Name: "proxy_url", Type: s"tftypes.String", Description: "URL to the proxy to be used for all API requests", Optional: true, ...},
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		&{
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		Name: "tls_server_name",
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		Type: s"tftypes.String",
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		Description: strings.Join({
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64: - 		"P",
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64: + 		"Server name p",
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		"assed to the server for SNI and is used in the client to check s",
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		"erver certificates against.",
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		}, ""),
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		Required: false,
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		Optional: true,
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		... // 4 identical fields
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		},
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		&{Name: "token", Type: s"tftypes.String", Description: "Token to authenticate an service account", Optional: true, ...},
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		&{Name: "username", Type: s"tftypes.String", Description: "The username to use for HTTP basic authentication when accessing"..., Optional: true, ...},
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		})),
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		BlockTypes:  {&{TypeName: "exec", Block: &{Attributes: Inverse(cmpopts.SortSlices, []*tfprotov5.SchemaAttribute{&{Name: "api_version", Type: s"tftypes.String", Required: true}, &{Name: "args", Type: s"tftypes.List[tftypes.String]", Optional: true}, &{Name: "command", Type: s"tftypes.String", Required: true}, &{Name: "env", Type: s"tftypes.Map[tftypes.String]", Optional: true}})}, Nesting: s"LIST", MaxItems: 1}, &{TypeName: "experiments", Block: &{Attributes: {&{Name: "manifest_resource", Type: s"tftypes.Bool", Description: "Enable the `kubernetes_manifest` resource.", Optional: true, ...}}, Description: "Enable and disable experimental features."}, Nesting: s"LIST", MaxItems: 1}},
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		Description: "",
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   		... // 2 identical fields
2022-03-21T12:07:35.594-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   	},
2022-03-21T12:07:35.595-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:   }
2022-03-21T12:07:35.595-0400 [DEBUG] provider.terraform-provider-kubernetes_9.9.9_darwin_amd64:
```